body/stretch_body/robot_params.py

```diff
@@ -25,7 +25,7 @@
                     "class": "logging.FileHandler",
                     "level": "DEBUG",
                     "formatter": "default_file_formatter",
-                    "filename": hello_utils.get_stretch_directory('log/') + 'stretchbody_{0}.log'.format(
+                    "filename": hello_utils.get_stretch_directory('log/stretch_body_logger/') + 'stretchbody_{0}.log'.format(
```

Comment:
I'd advocate for `hello_utils.get_stretch_directory()` to have an optional argument, `create_if_doesnt_exist`, so we can use new directories like this without worrying about creating them.